test/martha/martha_v3.test.js

```diff
@@ -525,6 +525,31 @@ test.serial('martha_v3 parses BDC staging response correctly', async (t) => {
     t.is(actualProvider, expectedProvider);
 });
 
+test.serial('martha_v3 parses uppercase BDC staging response correctly', async (t) => {
```

Comment:
Except for `712c` -> `712C`, is this exactly the same as the test above? I don't think we'd lose any meaningful coverage if we test with just `712C`.

Reply:
Happy to switch to only testing `712C` capitalized 👍